ui/admin/tests/acceptance/targets/credential-sources-test.js

```diff
@@ -160,6 +160,16 @@ module('Acceptance | targets | credential-libraries', function (hooks) {
     assert.equal(findAll('tbody tr').length, count - 1);
   });
 
+  test('cannot remove credential libraries without proper authorization', async function (assert) {
```

Comment:
should this be for `credential stores`?

Reply:
No.  A target's "credential sources" refers to credential libraries.  In the future it might also allow one-off credentials (one level below library).